pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md

```diff
@@ -8,6 +8,40 @@ excerpt: Workspace Catalog Applications
 <!-- TODO: settle down the link target to custom applications -->
 Catalog applications are any third-party or open source applications that appear in the Catalog. These applications can be provided by D2iQ for use in your environment.
 
+## Installing the workspace catalog via the CLI
+
+1.  Set the `WORKSPACE_NAMESPACE` environment variable to the name of your workspace’s namespace:
+
+    ```bash
+    export WORKSPACE_NAMESPACE=<workspace namespace>
+    ```
+
+1.  Create the `GitRepository`:
+
+    ```sh
+    kubectl apply -f - <<EOF
+    apiVersion: source.toolkit.fluxcd.io/v1beta1
+    kind: GitRepository
+    metadata:
+      name: dkp-catalog-applications
+      namespace: ${WORKSPACE_NAMESPACE}
+      labels:
+        kommander.d2iq.io/gitapps-gitrepository-type: catalog
+    spec:
+      interval: 1m0s
+      ref:
+        branch: main
+      timeout: 20s
+      url: https://github.com/mesosphere/dkp-catalog-applications  
+    EOF
+    ```
+
+1.  You should now see the workspace catalog `Apps` available in the UI and CLI
```

Comment:
```suggestion
1.  You should now see the workspace catalog `Apps` available in the UI, and in the CLI using `kubectl`:
```

Reply:
good call, fixed in ac0b90a0140ab050ffc007c599d0216ff412e534!

Reply:
👍